### PR TITLE
chore: add asyncapi-tool.yaml file

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

- add `.asyncapi-tool.yaml` file to show the ServerAPI as tool in the `asyncapi.com/tools` dashboard.

cc @derberg

### CODE REVIEW

**Review by BOLT04 — APPROVED** [review]
looks good to me 👍 
It seems this `asyncapi-tool.yaml` is a new way to automatically have tools on the website. Found this PR with more info about it https://github.com/asyncapi/website/pull/1135 😃 
we can merge if you want @magicmatatjahu 👍

File: .asyncapi-tool.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: Server API providing official AsyncAPI tools.
+links:
+  websiteUrl: https://api.asyncapi.com/v1
+  docsUrl: https://api.asyncapi.com/v1/docs
+filters:
+  language: JavaScript
+  technology:
+    - Express.js
+    - TypeScript
+  categories:
```
Comment (derberg) [inline .asyncapi-tool.yaml:11]: imho API is enough. 

category is a grouping parameter, `server` group will not really bring much value for person looking for tools

File: .asyncapi-tool.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+title: AsyncAPI Server API
+description: Server API providing official AsyncAPI tools.
+links:
+  websiteUrl: https://api.asyncapi.com/v1
+  docsUrl: https://api.asyncapi.com/v1/docs
+filters:
+  language: JavaScript
```
Comment (derberg) [inline .asyncapi-tool.yaml:7]: please remove, unless it is possible to integrate on a language level

File: .asyncapi-tool.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+title: AsyncAPI Server API
+description: Server API providing official AsyncAPI tools.
```
Comment (derberg) [inline .asyncapi-tool.yaml:2]: since it is the same as repo description, I suggest to remove it

**Comment by magicmatatjahu** [comment]
@BOLT04 Yeah, I forgot to share info about details in the website repo 😅 Thanks!

File: .asyncapi-tool.yaml (hunk 1 above)
Comment (magicmatatjahu) [reply .asyncapi-tool.yaml:11]: As I see, we don't have `api` category. Should I add this, or someone else (like you)?

Comment (derberg) [reply .asyncapi-tool.yaml:9]: you can add it, do it for `editor` too

**Review by derberg — APPROVED** [review]

**Review by BOLT04 — DISMISSED** [review]

**Comment by BOLT04** [comment]
not sure why checks aren't running 😢

File: .asyncapi-tool.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+title: AsyncAPI Server API
+links:
+  websiteUrl: https://api.asyncapi.com/v1
+  docsUrl: https://api.asyncapi.com/v1/docs
+filters:
+  technology:
+    - Node js
+    - TypeScript
+  categories:
+    - api
```
Comment (derberg) [inline .asyncapi-tool.yaml:10]:
```suggestion
title: AsyncAPI Server API
links:
  websiteUrl: https://api.asyncapi.com/v1
  docsUrl: https://api.asyncapi.com/v1/docs
filters:
  technology:
    - Node.js
    - TypeScript
  categories:
    - api
```

suggesting the change only to get this merged and try to trigger CI runs again

**Review by derberg — APPROVED** [review]

**Pull request merged**